doc/api/deprecations.md

```diff
@@ -2230,6 +2230,20 @@ like `dns.lookup(false)` due to backward compatibility.
 This behavior is undocumented and is thought to be unused in real world apps.
 It will become an error in future versions of Node.js.
 
+<a id="DEP0XXX"></a>
+### DEP0XXX: setting the TLS ServerName to an IP address
+<!-- YAML
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/REPLACEME
+    description: Runtime deprecation.
+-->
+
+Type: Runtime
+
+Setting the TLS ServerName to an IP address is not permitted by RFC6066.
```

Comment:
replace replace `RFC6066` with `[RFC 6066][]`

And add a link reference to RFC 6066 at the bottom.